android/src/main/java/io/mosip/tuvali/rnModule/RNVerifierModule.kt

```diff
@@ -0,0 +1,42 @@
+package io.mosip.tuvali.rnModule
+
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+import io.mosip.tuvali.verifier.Verifier
+
+class RNVerifierModule(reactContext: ReactApplicationContext) :
+  ReactContextBaseJavaModule(reactContext) {
+  private val eventEmitter: RNEventEmitter = RNEventEmitter(reactContext)
+  private val verifier: Verifier = Verifier(reactContext)
```

Comment:
change type to Interface instead of concrete type in declaration

Reply:
Done.

To understand, Any class that we are passing between layers, we should try to use interface right?

Reply:
An interface helps to loosely couple RNVerifierModule with Verifier implementation. Ideal way to do this, we can inject Verifier Object through constructor of RNVerifierModule

E.g: From RNTuvali we can call like this `RNVerifierModule(reactContext, verifierObj)` and the declaration of RNVerifierModule will look this `class RNVerifierModule(reactContext: ReactApplicationContext, verifier: IVerifier)`

This way, RNVerifierModule can take any implementation of Verifier object as long as it satisfies the `IVerifier` interface. Also makes RNVerifierModule testable.